analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/SymbolicRuleCheck.cs

```diff
@@ -49,14 +49,29 @@ public void Init(SonarAnalysisContext sonarContext, SonarSyntaxNodeReportingCont
     protected void ReportIssue(IOperationWrapperSonar operation, params object[] messageArgs) =>
         ReportIssue(operation.Instance, messageArgs);
 
-    protected void ReportIssue(IOperation operation, params object[] messageArgs) => ReportIssue(operation, null, messageArgs);
+    protected void ReportIssue(IOperation operation, params object[] messageArgs) =>
+        ReportIssue(operation, null, messageArgs);
```

Comment:
```suggestion
    protected void ReportIssue(IOperation operation, params object[] messageArgs) => ReportIssue(operation, null, messageArgs);
```
This seems simple enough to keep it in one line.

Reply:
We only keep a lambda in one line only if it returns something as simple as a property or a literal.
For anything else (in this case method call) we put it in a new line.

See last point [here](https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md#multi-line-statements).